File: e-mail-bot/api_test.py
import json
import requests
from requests.auth import HTTPBasicAuth

def post_request():
    url = "{{API_URL}}/get_qr/"
    with open('example.json', 'r') as file:
        data = json.load(file)

    headers = {
        "Content-Type": "application/json"
    }
    auth = HTTPBasicAuth('api_user', 'V7INZ7e_rO')
    response = requests.post(url, json=data, headers=headers, auth=auth, timeout=60)
    return response

response = post_request()
print(response.json())

File: e-mail-bot/api_test.php
<?php
function post_request() {
    $url = "http://147.45.105.97:8000/get_qr/";
    $data = json_decode(file_get_contents('example.json'), true);

    $headers = [
        "Content-Type: application/json"
    ];

    $auth = base64_encode('api_user:V7INZ7e_rO');

    $options = [
        'http' => [
            'header'  => [
                "Content-Type: application/json",
                "Authorization: Basic $auth"
            ],
            'method'  => 'POST',
            'content' => json_encode($data),
            'timeout' => 60
        ]
    ];

    $context  = stream_context_create($options);
    $response = file_get_contents($url, false, $context);

    return $response;
}

$response = post_request();
echo $response;
?>
